src/app/components/viewer/viewer.component.ts

```diff
@@ -473,4 +479,16 @@ export class AppViewerComponent implements OnInit, OnDestroy {
       this.containersSkipNavigation.includes(className)
     );
   }
+
+  @HostListener('document:keydown', ['$event'])
+  handleKeyboardEvent(event: KeyboardEvent) {
```

Comment:
This will prevent the keyboard move the cursor in a textbox which is not desired

Reply:
Currently i have created ADF pr to avoid it. (https://github.com/Alfresco/alfresco-ng2-components/pull/5562).